securesystemslib/signer/_aws_signer.py

```diff
@@ -101,65 +102,95 @@ def import_(cls, aws_key_id: str, local_scheme: str) -> Tuple[str, Key]:
         Raises:
             UnsupportedAlgorithmError: If the AWS KMS signing algorithm is
             unsupported.
-            BotoCoreError: Errors from the botocore.exceptions library.
+            BotoCoreError: Errors from the botocore library.
             ClientError: Errors related to AWS KMS client.
         """
         if AWS_IMPORT_ERROR:
             raise UnsupportedLibraryError(AWS_IMPORT_ERROR)
 
-        client = boto3.client("kms")
-        request = client.get_public_key(KeyId=aws_key_id)
-        kms_pubkey = serialization.load_der_public_key(request["PublicKey"])
-
-        public_key_pem = kms_pubkey.public_bytes(
-            encoding=serialization.Encoding.PEM,
-            format=serialization.PublicFormat.SubjectPublicKeyInfo,
-        ).decode("utf-8")
         try:
-            keytype = cls._get_keytype_for_scheme(local_scheme)
-        except KeyError as e:
-            raise exceptions.UnsupportedAlgorithmError(
-                f"{local_scheme} is not a supported signing algorithm"
-            ) from e
+            client = boto3.client("kms")
+            request = client.get_public_key(KeyId=aws_key_id)
+            kms_pubkey = serialization.load_der_public_key(request["PublicKey"])
+            keytype = cls._get_keytype_from_aws_response(request)
+
+            if local_scheme:
+                try:
+                    scheme = cls._get_aws_signing_algo(local_scheme)
```

Comment:
This is the wrong scheme to be used for SSlibKey below.

I ran my [localstack test from RSTUF](https://github.com/repository-service-tuf/repository-service-tuf-worker/pull/452) against your PR and it failed with this error message:

```
.tox/local-aws-kms/lib/python3.10/site-packages/securesystemslib/signer/_aws_signer.py:67: in __init__
    self.aws_algo = self._get_aws_signing_algo(self.public_key.scheme)
...
E       KeyError: 'RSASSA_PSS_SHA_256'
```

Reply:
It doesn't look like you fixed this bug. When I pass `"rsassa-pss-sha256"` to `import_`, it returns an `SSlibKey` with  `"RSASSA_PSS_SHA_256"` as `scheme`. Later, when I pass that key to  `AWSigner.__init__`, I get the above error.

Reply:
ah, thanks for the heads up. i think that's fixed now.